### ISSUE TITLE
EmptyEpsilon refuses to run in fullscreen on multiple monitors

### ISSUE THREAD

**Issue by JordanLongstaff** [daid/EmptyEpsilon#542]
I have a computer with two monitors. I can run one instance in fullscreen on one of the monitors. When I try to run a second instance, it insists on opening up on the same monitor as the first instance. Even if I switch it to window, move the window to the other monitor and then go back to fullscreen, it switches itself back to the same monitor as the other instance.

**Comment by k9lego** [comment]
Unfortunately there is no solution to that issue. The only possible solution is a third party utility that makes it "windowed" full-screen.

**Comment by JordanLongstaff** [comment]
If that's what you have to do, I'm all for it. Artemis has no problem doing that.

**Comment by k9lego** [comment]
Yeah, the main issue is with the underlying program. Here's a discussion that I had in the EE Forums.
http://bridgesim.net/discussion/330/full-screen-issues

**Comment by eode** [comment]
I don't know about Linux, but on Windows:

* The Primary Display can be set (select the monitor in display settings, and choose "set as primary display" or similar option)
* Empty Epsilon changes to the current primary display any time it is full-screened -- and stays there.

Since that's the case, you can do this:

1. Set Display 1 to be the Primary Display
2. Full-screen Empty Epsilon (first instance)
3. Set Display 2 to be the Primary Display
4. Full-screen Empty Epsilon (second instance)

...repeat as needed.

**Comment by oznogon** [comment]
For: https://github.com/daid/SeriousProton/blob/ac7f04b65e730a4bdee61b2ed85843ed57088fba/src/windowManager.cpp#L93-L96

Would having an option to set the resolution and use `sf::Style::None`/a borderless window instead of `sf::Style::Fullscreen` be a workaround:

```
    window.create(sf::VideoMode(windowWidth, windowHeight, 32), WINDOW_TITLE, sf::Style:: None, context_settings);
```

OS controls could then be used to move a borderless window to another monitor as a clunky workaround.
